app/Console/UpdateEventsAttendanceMode.php

```diff
@@ -45,24 +47,19 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $query = $this->askForQuery($input, $output);
         $attendanceMode = $this->askAttendanceMode($input, $output);
 
+        $locationId = new LocationId(LocationId::NIL_LOCATION);
+        if (!$attendanceMode->sameAs(AttendanceMode::online())) {
+            $locationId = $this->askForPhysicalLocation($input, $output);
```

Comment:
Wouldn't it be prudent to prevent users from giving the online uuid `00000000-0000-0000-0000-000000000000`? Now this command is able to create events with "attendanceMode": "offline" and "locationId": "00000000-0000-0000-0000-000000000000".

Reply:
Good remark! That is indeed guarded inside the request handler https://github.com/cultuurnet/udb3-silex/blob/814416b08c4980f554b2eea3fde4762d132c94a4/src/Http/Event/UpdateAttendanceModeRequestHandler.php#L78 but not inside the CLI command

Reply:
This is fixed now.